openwisp_utils/admin_theme/static/admin/js/menu.js

```diff
@@ -1,21 +1,52 @@
 'use strict';
-(function () {
-  window.addEventListener('load', function(){
-    var nav = document.getElementById('main-menu'),
-        container = document.getElementById('container'),
-        header = document.getElementById('header'),
-    setHeightToMenu = function() {
-      var windowHeight = window.innerHeight,
-          pageHeight = document.body.scrollHeight || container.offsetHeight,
-          height = windowHeight;
-      if (pageHeight > windowHeight) {
-        height = pageHeight;
-      }
-      if (!nav) { return; }
-      nav.style.height = (height - header.offsetHeight) + 'px';
-      nav.style.marginTop = header.offsetHeight + 'px';
-    };
-    setHeightToMenu();
-    window.addEventListener('resize', setHeightToMenu);
-  });
+const owContainer = document.getElementById('container');
+const owMenu = document.getElementById("menu");
+const owMainContent = document.getElementById("main-content");
+const owMenuToggle = document.querySelector('.menu-toggle');
```

Comment:
Please check if the vars are not `null` when you want to work with them like in line 29, 33 ...
There are cases, where no Menu will be rendered.

The exceptions are not dramatic, but will pollute the console log unnecessarily.